script.module.kodi-six/libs/kodi_six/utils.py

```diff
@@ -86,13 +90,12 @@ def patch_object(obj):
         obj = _wrap_class(obj)
         for memb_name, member in inspect.getmembers(obj):
             # Do not patch special methods!
-            if (inspect.ismethoddescriptor(member) and
-                    not memb_name.endswith('__')):
+            if (inspect.ismethoddescriptor(member) and not memb_name.endswith('__')):
                 setattr(obj, memb_name, encode_decode(member))
     return obj
 
 
-class ModuleWrapper(object):
+class ModuleWrapper:  # pylint: disable=too-few-public-methods
```

Comment:
Why an old-style class in Python 2?

Reply:
Because (believe it or not) the Python 3 pylint complains about these useless new-style classes. And Python 2 pylint complains about old-style-classes, so I favored the Python 3 checks in this case and ignored the Python 2 checks.

It is not ideal, I know, but there is no way around this. My preference is to prefer Python 3 checks over Python 2 checks. Feel free to play with this yourself and test on both Python 2 and Python 3.